torchvision/prototype/transforms/_augment.py

```diff
@@ -247,6 +247,7 @@ def _copy_paste(
         xyxy_boxes = masks_to_boxes(masks)
         # TODO: masks_to_boxes produces bboxes with x2y2 inclusive but x2y2 should be exclusive
         # we need to add +1 to x2y2. We need to investigate that.
+        # datumbox: I had a look on other reference implementations and I see a similar +1 to make it exclusive. I think we can keep it.
```

Comment:
cc @vfdev-5

Reply:
Anything to do with this comment or we can resolve this conversation ?

Reply:
Yes remove the TODO and just leave the comment. I think the +1 needs to stay for the reason you mention. :)

Reply:
https://github.com/pytorch/vision/pull/6498